Improve Backend Documentation (#670)
kinvolk/lokomotive: docs/configuration-reference/backend/s3.md

## Prerequisites

* AWS S3 bucket to be used should already be created. This can be done through the following aws CLI command:

> ```

Reviewer: ```suggestion
```sh
```

aws s3 mb s3://kinvolk-lokomotive-test --region=eu-central-1
```

* DynamoDB table to be used for state locking should already be created. The table must have a primary key named LockID.

> This can be established with the `aws dynamodb create-table` command:

Reviewer 2: I find the word "established" weird here. Also you can apply the more direct style and be more consistent with the previous point. How about:

```suggestion
You can create it with the following AWS CLI command:
```

> 
> ```

Reviewer: ```suggestion
```sh
```
